holoviews/operation/datashader.py

```diff
@@ -168,6 +179,13 @@ def get_agg_data(cls, obj, category=None):
             df = paths[0]
         if category and df[category].dtype.name != 'category':
             df[category] = df[category].astype('category')
+
+        for d in (x, y):
+            if df[d].dtype.kind == 'M':
+                param.warning('Casting %s dimension data to integer '
+                              'datashader cannot process datetime data ')
```

Comment:
You can add a datetime formatter to your axis and it will work. I think it's fine with the warning.